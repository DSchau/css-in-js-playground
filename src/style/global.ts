export const GLOBAL = `
  html, body {
    font-family: sans-serif;
  }

  html, body, #root {
    height: 100%;
  }

  body {
    overflow: hidden;
  }

  .wf-active {
    font-family: 'Bitter', sans-serif;
  }
`;
