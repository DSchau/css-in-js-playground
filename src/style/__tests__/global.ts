import { GLOBAL } from '../global';

test('it matches snapshot', () => {
  expect(GLOBAL).toMatchSnapshot();
});
