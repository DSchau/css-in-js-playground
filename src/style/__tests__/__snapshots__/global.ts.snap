// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
"
  html, body {
    font-family: sans-serif;
  }

  html, body, #root {
    height: 100%;
  }

  body {
    overflow: hidden;
  }

  .wf-active {
    font-family: 'Bitter', sans-serif;
  }
"
`;
